packages/tools/lib/monitor-postcss/index.js

```diff
@@ -0,0 +1,65 @@
+const chokidar = require("chokidar");
+const commandLineArgs = require("command-line-args");
+const psList = require("ps-list");
+const fkill = require("fkill");
+const { exec } = require("child_process");
+
+const options = commandLineArgs([
+	{ name: "srcFiles", type: String },
+	{ name: "packageName", type: String },
+]);
+const packageName = options.packageName;
+
+const restartPostcss = async () => {
+	let retryCount = 5;
+	let processes;
+
+	while (retryCount) {
+		try {
+			console.log("Getting processes list...");
+			processes = await psList();
+			retryCount = 0;
+		} catch (e) {
+			console.log("failed, retrying...");
+			retryCount--;
+		}
+	}
+
+	const process = processes.find(p => p.cmd.includes("postcss") && p.cmd.endsWith(`-w --packageName=${packageName}`));
+	if (process) {
+		await fkill(process.pid);
+
+		exec(`npx nps build.styles.components`, (err, stdout, stderr) => {
```

Comment:
isn't the watch below sufficient because a new start will see the newly added files?

Reply:
It turns out no, that's why probably we have first `build.styles.components` and only then `watch.styles.components`. The `-w` flag does not execute the initial action.